src/main/java/teammates/ui/template/FeedbackQuestionEditForm.java

```diff
@@ -0,0 +1,160 @@
+package teammates.ui.template;
+
+import java.util.List;
+
+import teammates.common.datatransfer.FeedbackQuestionAttributes;
+import teammates.common.util.Url;
+
+/**
+ * Data model for adding/editing a single question 
+ *
+ */
+public class FeedbackQuestionEditForm {
+    //TODO switch the rest of the Links to use Url if they are links
+    private Url actionLink;
+    
+    private String courseId;
+    private String feedbackSessionName;
+    
+    private String questionNumberSuffix;
+    
+    // Used for adding a new question
+    private String questionTypeOptions;
+    private Url doneEditingLink;
+    
+    private int numOfQuestionsOnPage;
```

Comment:
Should this belong to `FeedbackQuestionEditForm`?
Wouldn't it be the same for every one of them in the `List<FeedbackQuestionEditForm>` in `*PageData`?
And on that note, is it any different from `qnForms.size()` from `*PageData`?